alcs/acs2/Classifier.py

```diff
@@ -71,7 +77,8 @@ def copy_from(cls, old_cls, time):
         :return: new classifier
         """
         new_cls = cls(
-            condition=old_cls.condition,
+            condition=Condition(old_cls.condition),
+            #         ^^^^^^^^^ copy it, not just pass the reference!
```

Comment:
No comments pls.

Reply:
Yes, it was one of the sources - it was copying just a reference to an old `Condition` which may have changed due to mutation or crossover. Tests I wrote for that - `ClassifierTest.test_copy_from_and_mutate_does_not_influence_another_condition()` and `ClassifierTest.test_copy_from_and_mutate_does_not_influence_another_effect()` were failing.
Will remove the comment.